modules/vsphere-anti-affinity.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-vsphere.adoc
+// * installing/installing_vmc/installing-restricted-networks-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations.adoc
+// * installing/installing_vmc/installing-vmc.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-user-infra.adoc
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc
+// * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+:_content-type: PROCEDURE
+[id="anti-affinity-vsphere_{context}"]
+= Configure vSphere DRS anti-affinity rules for control plane nodes
+
+{product-title} Distributed Resource Scheduler (DRS) anti-affinity rules can be configured to support higher availability of OpenShift Control Plane nodes so that the vSphere Virtual Machines for these nodes are not scheduled to the same vSphere Host.
```

Comment:
Can we cross-link to the doc below for reference? 

https://github.com/openshift/openshift-docs/pull/51056

Reply:
@bostrt I can't install a cross-link in a module (this keeps modules portable, which this one will be once I move it to UPI section from IPI section). However there are other ways we can refer to other places in the documentation. I'll look to see what I'm able to do.

Reply:
(and BTW, @bostrt I know that seems...oddly primitive in this day and age. It's because of the post-processing for the customer portal. I dislike it myself.)

Reply:
Hey @bostrt the doc produced by that PR already appears "above" and as a prerequisite in all the installation sections, including the one that this section is going into. Because of that, may I avoid mentioning it in our PR?

Reply:
Thank you sir....